packages/docusaurus-plugin-openapi-docs/src/markdown/createRequestHeader.ts

```diff
@@ -5,6 +5,20 @@
  * LICENSE file in the root directory of this source tree.
  * ========================================================================== */
 
+import { create } from "./utils";
+
 export function createRequestHeader(header: string) {
-  return `## ${header}\n\n`;
+  return [
+    create(
+      "Heading",
+      {
+        children: header,
+        id: header.replace(" ", "-").toLowerCase(),
```

Comment:
What is the reason for the replace here?

Reply:
to account for the case of generating an `id` based on `header` when `Header` has spaces

https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/packages/docusaurus-plugin-openapi-docs/src/openapi/openapi.ts#L88